tribe-ext-pdf-tickets.php

```diff
@@ -0,0 +1,1645 @@
+<?php
+/**
+ * Plugin Name:       Event Tickets Extension: PDF Tickets
+ * Description:       Event Tickets' RSVP, Tribe Commerce PayPal, WooCommerce, and/or Easy Digital Downloads ticket emails will become PDF files saved to your Uploads directory and then get attached to the ticket emails.
+ * Version:           1.1.0
+ * Extension Class:   Tribe__Extension__PDF_Tickets
+ * GitHub Plugin URI: https://github.com/mt-support/tribe-ext-pdf-tickets
+ * Author:            Modern Tribe, Inc.
+ * Author URI:        http://m.tri.be/1971
+ * License:           GPL version 2
+ * License URI:       https://www.gnu.org/licenses/gpl-2.0.html
+ * Text Domain:       tribe-ext-pdf-tickets
+ */
+
+// Do not load unless Tribe Common is fully loaded and our class does not yet exist.
+if (
+	class_exists( 'Tribe__Extension' )
+	&& ! class_exists( 'Tribe__Extension__Example' )
```

Comment:
Should this be `! class_exists( 'Tribe__Extension__PDF_Tickets' )`?

Reply:
yar! fixed.